ts/IterableInboxMessageListItem.tsx

```diff
@@ -0,0 +1,77 @@
+"use strict"
+import React, { Component, useState } from "react"
+import { 
+   View, 
+   Text, 
+   StyleSheet 
+} from "react-native"
+
+type MessageListItemProps = {
+   index: number,
+   message: {[key: string]: any},
```

Comment:
you may want to create a specific type for a message rather than relying on `any`; this will help safe-guard you in the future.

Reply:
I agree. I created a message type file and imported the custom message type to all files that use the message type.